flow/flow.go

```diff
@@ -135,12 +135,18 @@ func FLowsFromSFlowSample(ft *FlowTable, sample *layers.SFlowFlowSample, probePa
 	for _, rec := range sample.Records {
 
 		/* FIX(safchain): just keeping the raw packet for now */
-		record, ok := rec.(layers.SFlowRawPacketFlowRecord)
-		if !ok {
-			logging.GetLogger().Critical("1st layer is not SFlowRawPacketFlowRecord type")
+		switch rec.(type) {
+		case layers.SFlowRawPacketFlowRecord:
```

Comment:
hmm in C you print a log and continue
in Go you breaking out the switch

At least put the general case ( case layers.SFlowRawPacketFlowRecord: ) at the end of the switch